pyartcd/pyartcd/pipelines/gen_assembly.py

```diff
@@ -204,6 +207,16 @@ async def _create_or_update_pull_request(self, assembly_definition: OrderedDict)
             else:
                 pull_number = existing_prs.items[0].number
                 result = api.pulls.update(pull_number=pull_number, title=title, body=body)
+
+            if self.auto_trigger_build_sync:
+                self._logger.info("Triggering build sync")
+                build_version = self.group.split("-")[1]  # eg: 4.14 from openshift-4.14
```

Comment:
Might be prettier to change pipeline param to `version` and change [pyartcd invocation](https://github.com/openshift-eng/aos-cd-jobs/blob/master/jobs/build/gen-assembly/Jenkinsfile#L122) to:
```
cmd += [
                "gen-assembly",
                "--data-path", params.DOOZER_DATA_PATH,
                "--version", "$params.BUILD_VERSION",
                "--assembly", params.ASSEMBLY_NAME,
            ]
```
Then, we would create fields such as 
```
self.version = version
self.group = f'openshift-{version}'
```
without needing to change anything else

Reply:
Makes sense. Making the change

Reply:
Fixed in https://github.com/openshift-eng/aos-cd-jobs/pull/3605/commits/43560fc58fd4e127fa655bd82dda758d8965b529